draft-jmiller-json-web-proof.md

```diff
@@ -121,44 +98,27 @@ For example:
 }
 ```
 
+Every JWP algorithm must include a digest method that is used to generate a hash of the base64url serialized protected header.  The protected header cannot be selectively disclosed and the digest value MUST be included in all proof values.
```

Comment:
This sounds to me like https://datatracker.ietf.org/doc/html/rfc7638 but applying to JWP protected headers.

We should beware of serialization ordering issues with objects.... 

This digest would function as a correlation identifier for any derived proof.

Reply:
Another great topic for the next call...

Reply:
Excellent point @OR13, this definitely needs some language around that, with probably more detail in a security considerations.

The protected header SHOULD be minimal and static across all usages of a given key, and static as in simply re-using the digested version.  It's only real purpose is to contain signals of how to process the JWP, key id, etc.